src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -3340,16 +3340,4 @@ namespace winrt::TerminalApp::implementation
             }
         }
     }
-
-    // -------------------------------- WinRT Events ---------------------------------
-    // Winrt events need a method for adding a callback to the event and removing the callback.
-    // These macros will define them both for you.
-    DEFINE_EVENT_WITH_TYPED_EVENT_HANDLER(TerminalPage, TitleChanged, _titleChangeHandlers, winrt::Windows::Foundation::IInspectable, winrt::hstring);
```

Comment:
Are we able to kill the `DEFINE_EVENT_WITH_TYPED_EVENT_HANDLER` macro now?

Reply:
unsure if that's the last one, I can check though

Reply:
gottem all